### PR TITLE
first draft support for 3.9

### PR DESCRIPTION
**Pull request by richm**
first draft support for 3.9
@sradco

### CODE REVIEW

File: centos7-viaq.repo [hunk 1]
```diff
@@ -40,11 +40,17 @@ gpgcheck=0
...
 
 [centos-openshift-origin39-candidate]
 name=CentOS OpenShift Origin39 Candidate
 baseurl=http://cbs.centos.org/repos/paas7-openshift-origin39-candidate/x86_64/os
+enabled=1
+gpgcheck=0
+
+[centos-openshift-origin310-candidate]
+name=CentOS OpenShift Origin310 Candidate
+baseurl=http://cbs.centos.org/repos/paas7-openshift-origin310-candidate/x86_64/os
```
Comment (sradco) [inline centos7-viaq.repo]: Why add version 10 if you dont supply the ansible-inventory for it?
Can users use openshift-origin 3.10?

File: ansible-inventory-origin-39-aio [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ansible_connection=local
+openshift_release=v3.9
+openshift_logging_install_logging=true
+short_version=3.9
+openshift_deployment_type=origin
+openshift_master_identity_providers=[{'mappingMethod': 'lookup', 'challenge': 'true', 'login': 'true', 'kind': 'AllowAllPasswordIdentityProvider', 'name': 'allow_all'}]
+# bug - doesn't use correct tag
+openshift_logging_image_version=v3.9
+openshift_logging_elasticsearch_proxy_image_version=v1.0.0
+
```
Comment (sradco) [inline ansible-inventory-origin-39-aio:17]: I see you removed the following flags:
-openshift_hosted_logging_deploy=true
-openshift_image_tag=latest
-oreg_url=openshift/origin-${component}:latest
-openshift_logging_namespace=logging
-openshift_logging_es_cluster_size=1
-openshift_logging_image_prefix=docker.io/openshift/origin-
-deployment_type=origin
-openshift_logging_elasticsearch_proxy_image_prefix=docker.io/openshift/
-openshift_logging_elasticsearch_proxy_image_version=v1.1.0

Are they not needed anymore?

File: ansible-inventory-ocp-39-aio [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+openshift_logging_elasticsearch_storage_type=hostmount
+openshift_logging_elasticsearch_hostmount_path=/var/lib/elasticsearch
+
+[nodes]
+localhost storage=True openshift_node_labels="{'region': 'infra'}" openshift_schedulable=True
+
+[masters]
+localhost storage=True openshift_node_labels="{'region': 'infra'}" openshift_schedulable=True
+
+[etcd]
```
Comment (sradco) [inline ansible-inventory-ocp-39-aio:28]: How will this affect the installation? Is it required in the oVirt use case?

Comment (sradco) [reply ansible-inventory-ocp-39-aio:28]: I see you removed these flags:
-openshift_hosted_logging_deploy=true
-openshift_logging_namespace=logging
-openshift_logging_es_cluster_size=1
-openshift_logging_image_version=v3.6
-deployment_type=openshift-enterprise
-openshift_disable_check="package_version,docker_storage"

Are they not needed anymore?

Comment (richm) [reply ansible-inventory-ocp-39-aio:28]: > How will this affect the installation? Is it required in the oVirt use case?

not sure what you mean

File: ansible-inventory-origin-39-aio (hunk 2 above)
Comment (richm) [reply ansible-inventory-origin-39-aio:17]: see latest

File: ansible-inventory-ocp-39-aio (hunk 3 above)
Comment (richm) [reply ansible-inventory-ocp-39-aio:28]: see latest

File: centos7-viaq.repo (hunk 1 above)
Comment (richm) [reply centos7-viaq.repo]: no - removed 3.10

**Comment by richm** [comment]
@sradco if ok - I'm ready to merge this

**Comment by richm** [comment]
@sradco not sure why it worked with origin, but it isn't working with ocp without the network_manager playbook.

**Pull request merged**